docs/reST/ref/key.rst

```diff
@@ -181,6 +181,7 @@ that your key handling code works well on both pygame 1 and pygame 2.
       K_MENU                menu
       K_POWER               power
       K_EURO                Euro
+      K_AC_BACK             Android back button
```

Comment:
Is it Android specific? I thought those worked on keyboards too?

https://wiki.libsdl.org/SDL_ScancodeAndKeycode
> "AC Back" (the Back key (application control keypad)).

Reply:
I guess I'll have to try this with a flirc and also a RF multimedia remote. I'll report back later